Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -192,6 +201,9 @@ public static int GetRemainingHours(RoomRental_v1 room)
 
         public override void Update(DaggerfallEntityBehaviour sender)
         {
+            if (preventUpdate)
```

Comment:
Is there anything stopping this from working using the existing mechanism? In other words change this to:

``if (SaveLoadManager.Instance.LoadInProgress)``

Reply:
Yes that should work. Good I can remove that bool then.